data_summaries/Deita 10K.json

```diff
@@ -0,0 +1,59 @@
+{
+    "deita-10k": {
+        "Unique Dataset Identifier": "deita-10k",
+        "Dataset Name": "deita-10k",
+        "Paper Title": "What Makes Good Data for Alignment? A Comprehensive Study of Automatic Data Selection in Instruction Tuning",
+        "Dataset URL": "https://huggingface.co/datasets/hkust-nlp/deita-10k-v0",
+        "GitHub URL": "https://github.com/hkust-nlp/deita",
+        "Hugging Face URL": "https://huggingface.co/datasets/hkust-nlp/deita-10k-v0",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2312.15685",
+        "Semantic Scholar Corpus ID": 266551413,
+        "Collection": "Deita 10K",
+        "Collection URL": "https://huggingface.co/collections/hkust-nlp/deita-6569c198c174808d94cf5bd4",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Alignment",
+            "Language Modeling"
+        ],
+        "Text Sources": [],
+        "Format": [
+            "Zero-shot",
+            "Multi-turn Dialog"
+        ],
+        "Dataset Filter IDs": [
+            "ShareGPT",
+            "UltraChat",
+            "WizardLM"
+        ],
+        "Model Generated": [
+            "OpenAI ChatGPT",
+            "OpenAI Playground"
+        ],
+        "Derived from Datasets": [
+            "ShareGPT",
+            "UltraChat",
+            "WizardLM"
+        ],
+        "Human Annotation": "Yes",
+        "Creators": [
+            "Beijing University of Posts and Telecommunications",
+            "Alibaba Group",
+            "The Hong Kong University of Science and Technology"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://huggingface.co/datasets/hkust-nlp/deita-10k-v0"
+            },
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://github.com/hkust-nlp/deita?tab=Apache-2.0-1-ov-file"
+            }
+        ],
+        "License Notes": "",
+        "License Verified By": "An"
+    }
+}
```

Comment:
For the "Creators" portion, there are no option for ShanghaiTech University and Meituan in the creator_groups.json file, so I have not yet added them.

Reply:
I think you need to add them in the `creator_groups.json` file and then in the `Creators` field.